implementations/rust/ockam/ockam_api/src/ui/terminal/mod.rs

```diff
@@ -326,6 +337,17 @@ impl<W: TerminalWriter + Debug> Terminal<W, ToStdOut> {
         self
     }
 
+    /// Return true if log messages are emitted to the console
+    fn logging_to_console(&self) -> bool {
+        self.logging_enabled && !self.logging_goes_to_file
+    }
+
+    /// Return true if we can write to stdout
+    /// We can write to stdout unless all the message are logged to the console
+    fn is_stdout_enabled(&self) -> bool {
```

Comment:
I'd rename these methods to `can_write_to_stdout/stderr`. It sounds more straightforward to me, what do you think?